packages/app-project/src/screens/ClassifyPage/components/YourStats/components/DailyClassificationsChart/DailyClassificationsChart.js

```diff
@@ -1,23 +1,112 @@
 import counterpart from 'counterpart'
-import PropTypes from 'prop-types'
+import { array, object, string } from 'prop-types'
 import React from 'react'
+import { withTheme } from 'styled-components'
+import { AxisBottom, AxisLeft } from '@vx/axis'
+import { Group } from '@vx/group'
+import { Bar } from '@vx/shape'
+import { scaleBand, scaleLinear } from '@vx/scale'
+import WidgetHeading from '../../../../../../shared/components/WidgetHeading'
 
 import en from './locales/en'
 
 counterpart.registerTranslations('en', en)
 
-function DailyClassificationsChart () {
+function DailyClassificationsChart ({ stats, projectName, theme }) {
+  const HEIGHT = 200
+  const PADDING = 20
+  const WIDTH = 300
+  const xScale = scaleBand({
+    rangeRound: [0, WIDTH],
+    domain: stats.map(stat => stat.longLabel),
+    padding: 0.1
+  });
+
+  const yScale = scaleLinear({
+    rangeRound: [HEIGHT - PADDING, 0],
+    domain: [0, Math.max(...stats.map(stat => stat.count))],
+    nice: true
+  });
+
+  function tickLabelProps () {
+    return {
+      dx: '-0.25em',
+      dy: '0.2em',
+      fill: theme.global.colors['light-4'],
```

Comment:
The labels and ticks are hard to read in light theme:

<img width="431" alt="Screen Shot 2019-09-20 at 2 32 40 PM" src="https://user-images.githubusercontent.com/5016731/65355271-31a66f80-dbb7-11e9-8c65-39e7b92aadba.png">

I would suggest adding a function to return to you the color to use so it can be set in the few places this is needed. I forgot to mention in the last review, we have light and dark theme color for text, so for consistency with the other components, let's use that: `(theme.dark) ? theme.global.colors.text.dark : theme.global.colors.text.light`

Reply:
I thought this was the same grey used in the design in #38 but it does look light.

Reply:
This graph begins on Sunday, not Monday. That’s wrong.

Reply:
Here's a screenshot for dark mode. The graph is an image in the InVision designs, so I can't inspect the colour but it isn't using the theme text colours.
![Screenshot showing the classification stats graph in dark mode.](https://user-images.githubusercontent.com/59547/65371723-7cc78d80-dc5e-11e9-8d19-5fbc0b62a278.png)

Reply:
I've set the axis colours to `light-3`/`light-5` for dark/light mode. It looks ok to me but might still be too light to read.

Reply:
I'm going to ignore the InVision designs and use the theme text colours for the axes. Let's see how those look.

Reply:
Thanks @eatyourgreens, sorry that wasn't the updated version with the correct colors; it's a screenshot from the light theme I'd intended to update and then it got missed. Mind posting a screenshot when this is ready?

Reply:
@beckyrother see the screenshots that Sarah posted from her laptop, later in this discussion.